src/GafferScene/Instancer.cpp

```diff
@@ -2624,19 +2655,33 @@ void Instancer::InstancerCapsule::render( IECoreScenePreview::Renderer *renderer
 	const IECore::ConstCompoundObjectPtr globals = m_instancer->inPlug()->globals();
```

Comment:
This comment would be better attached to the `InstancerCapsule::render()` line, but GitHub won't let me put it there. Minor thing, but I think my puny brain would cope better with this big function if there were comments with headings that broke it down into the "get the engines", "prepare the prototypes" and "output the instances" stages.

Reply:
Addressed.